components/twitter/actions/list-favorites/list-favorites.mjs

```diff
@@ -0,0 +1,53 @@
+import common from "../common.mjs";
+
+export default {
+  ...common,
+  key: "twitter-list-favorites",
+  name: "List Favorites",
+  description: "Return the most recent tweets liked by you or the specified user",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    screenName: {
+      propDefinition: [
+        common.props.twitter,
+        "screenName",
+      ],
+    },
+    count: {
+      propDefinition: [
+        common.props.twitter,
+        "count",
+      ],
+      description: "Specifies the number of records to retrieve. Must be less than or equal to 200; defaults to 20.",
+      optional: true,
+      default: 20,
+    },
+    includeEntities: {
+      propDefinition: [
+        common.props.twitter,
+        "includeEntities",
+      ],
+    },
+  },
+  async run() {
+    const {
+      screenName,
+      count,
+      includeEntities,
+    } = this;
+
+    const params = {
+      screenName,
+      count,
+      include_entities: includeEntities,
+    };
+    const tweets = await this.paginate(this.twitter.getLikedTweets.bind(this), params);
```

Comment:
I am wondering if we really need to paginate all the results. What do you think about to not paginate and use the pagination params as property and letting the user paginate it as it wishes?
I imagine that fetching all data can be problematic to the pipeline performance in some scenarios. Maybe the user would like to have more control over the pagination?

Reply:
We actually discussed this recently on Slack (https://pipedream-users.slack.com/archives/C01E5KCTR16/p1636481783045500) 
We definitely need the pagination, but I'll add in a `max` prop so the user can set it only retrieve a maximum number of results.

Reply:
I see, In this case, I think the `max` its a good idea :)